packages/contracts-bedrock/contracts/L1/MantlePortal.sol

```diff
@@ -191,7 +192,7 @@ contract MantlePortal is Initializable, ResourceMetering, Semver {
      */
     // solhint-disable-next-line ordering
     receive() external payable {
-        depositTransaction(msg.sender, msg.value, RECEIVE_DEFAULT_GAS_LIMIT, false, bytes(""));
+        depositTransaction(msg.sender, 0, RECEIVE_DEFAULT_GAS_LIMIT, false, bytes(""));
```

Comment:
ETH on L2 is an ERC20 token. So it is right the replace `msg.value` with 0. However, I think `bytes("")` is wrong here.

Reply:
This comment is still not resolved. Does you test cases cover this case?